net.astralvixen.geforceinfinity.yml

```diff
@@ -0,0 +1,68 @@
+id: net.astralvixen.geforceinfinity
+app-id: net.astralvixen.geforceinfinity
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.node20
+
+command: geforce-infinity
+
+finish-args:
+    - --share=network
+    - --share=ipc
+    - --device=dri
+    - --socket=x11
+    - --socket=wayland
+    - --filesystem=home
+    - --device=dri
+    - --socket=system-bus
+    # Add sandbox permissions
+    - --allow=devel
+    - --talk-name=org.freedesktop.Flatpak
+    - --env=ELECTRON_DISABLE_SANDBOX=1
+    - --filesystem=xdg-run/discord-ipc-0
+    - --filesystem=xdg-run/discord
+    # Add audio and microphone support
+    - --socket=pulseaudio
+    - --talk-name=org.freedesktop.portal.Desktop
+    - --talk-name=org.freedesktop.portal.Freedesktop
+    - --talk-name=org.freedesktop.portal.Audio
+    - --talk-name=org.freedesktop.portal.Camera
+
+modules:
+    - name: geforce-infinity
+      buildsystem: simple
+      build-commands:
+          - unzip -d "geforce-infinity" "geforce-infinity.zip"
+          - cp -r "geforce-infinity" /app
+          - install -Dm 644 "icon.png" "${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png"
+          - install -Dm 644 "${FLATPAK_ID}.metainfo.xml" "${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml"
+          - install -Dm 755 "geforce-infinity.sh" "/app/bin/geforce-infinity"
+          - install -Dm 644 "net.astralvixen.geforceinfinity.desktop" "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+
+      sources:
+          - type: file
+            url: https://github.com/AstralVixen/GeForce-Infinity/releases/download/1.1.1/GeForceInfinity-linux-1.1.1-x64.zip
+            dest-filename: geforce-infinity.zip
+            sha256: 6ec2756286250d17afb283b172d5591b026ce588322866fa83a920d3ab7447af
+            x-checker-data:
+                type: json
+                url: https://api.github.com/repos/AstralVixen/GeForce-Infinity/releases/latest
+                version-query: .tag_name | sub("^v"; "")
+                url-query: .assets[] | select(.name=="GeForceInfinity-linux-" + $version + "-x64.zip") | .browser_download_url
+
+          - type: script
+            dest-filename: geforce-infinity.sh
+            commands:
+                - |
+                    export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+                    exec zypak-wrapper /app/geforce-infinity/geforce-infinity "$@"
+          - type: file
+            path: icon.png
+          - type: file
+            path: net.astralvixen.geforceinfinity.desktop
+          - type: file
+            path: net.astralvixen.geforceinfinity.metainfo.xml
```

Comment:
<img width="1175" height="209" alt="image" src="https://github.com/user-attachments/assets/9d72b708-6d8f-4fef-96a2-ed99d25abfff" />

Reply:
what is unclear with "avoid including a copy in the submission"?